.github/workflows/cirque.yaml

```diff
@@ -57,6 +57,7 @@ jobs:
               with:
                 platform: linux
 
+            # TODO: Is what's being cached here actually compatible with a regular bootstrap?
```

Comment:
@andy31415 Do you know any details here? It's not clear to me whether the `bootstrap.sh` run inside that container has exactly the same effect as running it directly, but I would expect there must be some difference, otherwise this workflow could just re-use the regular bootstrap action?

Reply:
This should work as is for now since the `bootstrap-cache` action uses a different key from the cache step within the updated `bootstrap`, but it seems like a longer term solution would be to refactor this workflow to use a single cache step for both the bootstraps being performed here.

Reply:
I think the compile bootstrap (via bootstrap.sh) should be compatible. Apart from that, cirque also has its own "boostrap" which is some docker image build and that is unrelated.

I believe for the purposes of bootstrap.sh and code compilation we should be fine.

Reply:
Actually looking at the script, it seems everything is funneled through chip-build-cirque/run.sh so I am not clear why a local cache would help (probably it does not).

@yunhanw-google do you know if we need this boostrap cache here at all? If we run all commands inside a docker image, bootstrapping outside docker does not seem useful.

we may want to test out just deleting this caching and see if cirque still passes.

Reply:
I don't believe the image has the bootstrap results inside of it. 


@ksperling-apple does this build faster?

Reply:
> @ksperling-apple does this build faster?

Not doing the bootstrap if we have a cache hit seems to save around a minute per job, from looking at a small sample of runs.

Reply:
I'm going to resolve this conversation to let the PR merge, leaving cirque to be addressed in a follow-up.